traad/app.py

```diff
@@ -186,6 +186,18 @@ def remove_argument_view():
                                args['offset'])
 
 
+@app.post('/refactor/add_argument')
+def add_argument_view():
+    args = bottle.request.json
+    return standard_async_task(bottle.request.app.project.add_argument,
+                               args['index'],
```

Comment:
I think these are in the wrong order. `add_argument` seems to take path, offset, index, name, default, and value, in that order.